src/markup-components/components.ts

```diff
@@ -20,8 +20,8 @@ import {
   DividerProps,
   InputField as inputField,
   InputFieldProps,
-  LookupSecretSettingsProps,
-  LookupSecretSettingsSection as lookupSecretSettingsSection,
+  lookup_secretSettingsProps,
+  lookup_secretSettingsSection as lookup_secretSettingsSection,
```

Comment:
I think you caught a few too many occurrences

Reply:
whoops